src/app/chefs/ChefSchedule.jsx

```diff
@@ -18,14 +18,15 @@ class ChefSchedule extends React.Component {
     const { chefId } = this.props;
     axios.get('/api/chef/schedule', { params: { id: chefId } })
       .then((data) => {
-        this.setState({ schedule: data.data });
+        // save chefId to state to prevent unknown switching to different chefId number in render
```

Comment:
Can you leave a comment detailing how the code looks without this fix? If it takes more than 60s may not be worth it, but I'd like to see what you were first trying to do so that I can see if I can fix via ChefHome

Reply:
If I logged the chefId prop in multiple places (componentdidmount, render, etc). I could see that it was switching from chefId: 1 to chefId: 21.